src/Application/Commands/Business/ImportFromHttpRequest/ImportFromHttpRequestHandler.php

```diff
@@ -139,6 +140,22 @@ public function handle(ImportFromHttpRequestCommand $command)
             foreach ($admins as $admin) {
                 $admin->notify(new AdminNewBusinessReadyForReview($business, auth()->user()));
             }
+        } else if ($currentPublishingStatus !== PublishingStatus::HIDDEN && $newPublishingStatus === PublishingStatus::HIDDEN
+            && auth()->user()->isEditor()
+        ) {
+            // We have hidden the business as an Editor.  We want to alert regional admin.  At the moment
```

Comment:
Could we determine the region from the business and then only alert regional admins for that region?

Reply:
Good point.  Yes, pushed.